dashboard/app/helpers/users_helper.rb

```diff
@@ -82,6 +82,13 @@ def summarize_trophies(script, user = current_user)
       user_data[:trophies] = summarize_trophies(script, user)
     end
 
+    if script.professional_learning_course
+      unit_assignment = Plc::EnrollmentUnitAssignment.find_by(user: user, plc_course_unit: script.plc_course_unit)
+      if unit_assignment
+        user_data[:focusAreas] = unit_assignment.plc_module_assignments.map{ |a| a.plc_learning_module.stage.position }
```

Comment:
So it's a little weird to me that you are passing an array of focus area positions and calling them focus areas.

Suggest you either call the variable :focusAreaPositions. Or pass a list of stage names and change the way you are determining what is a focus area in course progress.